.storybook/preview.js

```diff
@@ -25,6 +25,9 @@ window.___navigate = (pathname) => {
 
 export const parameters = {
   actions: { argTypesRegex: "^on[A-Z].*" },
+  backgrounds: {
+    disable: true,
+  },
```

Comment:
To remove the dark mode toggle button that Storybook renders.

This is so that only the toggle from the Chakra addon is visible.

This can also be added on a separate PR if desired.